examples/service-clients/azure-client/todo-list/src/view.tsx

```diff
@@ -0,0 +1,205 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	CollaborativeInput,
+	CollaborativeTextArea,
+	SharedStringHelper,
+} from "@fluid-example/example-utils";
+import { useTree } from "@fluid-experimental/tree-react-api";
+import type { IFluidContainer } from "fluid-framework";
+import type { ISharedString, SharedString } from "fluid-framework/legacy";
+import React, { useEffect, useRef, useState } from "react";
+
+import { createTodoItem, type TodoListContainerSchema } from "./fluid.js";
+import type { TodoList, TodoItem } from "./schema.js";
+
+// eslint-disable-next-line import/no-unassigned-import
+import "./style.css";
+
+/**
+ * {@link TodoListAppView} input props.
+ */
+export interface TodoListAppViewProps {
+	readonly todoList: TodoList;
+	readonly container: IFluidContainer<TodoListContainerSchema>;
+}
+
+/**
+ * To-do list application view component.
+ */
+export const TodoListAppView: React.FC<TodoListAppViewProps> = (
+	props: TodoListAppViewProps,
+) => {
+	const { todoList, container } = props;
+
+	const [titleString, setTitleString] = useState<SharedString | undefined>();
```

Comment:
Should this be ISharedString?
```suggestion
	const [titleString, setTitleString] = useState<ISharedString | undefined>();
```